packages/ui-extensions/src/surfaces/admin/components/Grid/examples/col-span-layouts.html

```diff
@@ -0,0 +1,34 @@
+<s-stack gap="base">
+  <s-grid gridTemplateColumns="repeat(12, 1fr)" gap="base">
+    <s-grid-item gridColumn="span 12" gridRow="span 1">
+      <s-section>
+        <s-text>Full width field</s-text>
```

Comment:
```suggestion
        <s-text>Full-width field</s-text>
```

Adding a `-` for the `*-width` text here to match the example docs.

Reply:
solved